CHANGELOG.md

```diff
@@ -52,21 +64,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   where database_retention_ttl exceeds the realm's maximum datastream storage
   retention period, if set.
 
-## [1.1.2] - Unreleased
-### Added
-- [astarte_data_updater_plant] customize the number of consumer connections
-  to the AMQP broker using `DATA_UPDATER_PLANT_AMQP_CONSUMER_CONNECTION_NUMBER`
-  (defult: 10).
-
-### Fixed
-- Forward-port changes from release-1.0 (gracefully handle malformed
-  purge properties messages, fix message count metrics, fix typing issues
-  in payloads).
-- [astarte_data_updater_plant] do not leak consumer channels in corner cases.
-- [astarte_data_updater_plant] do not leak producer channels in corner cases.
-- [astarte_trigger_engine] Always treat event TTL for trigger policies in seconds,
-  not milliseconds.
-
```

Comment:
I think we want to leave this untouched.

We cannot be sure, but we might eventually tag a v1.1.2 on the release-1.1 branch.
So I would leave the responsibility of updating the changelog for 1.1.x versions to the release-1.1 branch.

Here we can just change the `## [Unreleased]` section to `## [1.2.0-rc.0] 31-05-2024`.
Looking at the git commit, it will still be clear that 1.2.0-rc.0 contains also changes from the unreleased 1.1.2 version.